projects/plugins/jetpack/extensions/blocks/ai-image/index.js

```diff
@@ -59,10 +71,7 @@ export const settings = {
 	},
 	edit,
 	/* @TODO Write the block editor output */
```

Comment:
```suggestion
```

Reply:
I don't want to lose @jeherve approval and really want to merge it because testing is pain. Can you remove that line in your PR?